src/app/data-model-interface/DataModel.h

```diff
@@ -38,10 +38,10 @@ namespace InteractionModel {
 ///     thread or equivalent
 ///   - class is allowed to attempt to cache indexes/locations for faster
 ///     lookups of things (e.g during iterations)
-class Model : public DataModelMetadataTree
+class DataModel : public DataModelMetadataTree
```

Comment:
So here is a question: why do we have `InteractionModel::DataModel`?  These are sort of orthogonal things....